OrbitGl/DataManager.h

```diff
@@ -39,6 +41,7 @@ class DataManager final {
   void set_selected_process(int32_t pid);
 
   [[nodiscard]] const ProcessData* GetProcessByPid(int32_t process_id) const;
+  [[nodiscard]] ProcessData* GetMutableProcessByPid(int32_t process_id);
```

Comment:
Where do you use that? In general, I think we should keep those accessors minimal. They basically allow us to completely shortcut the thread-safety measurements of DataManager (here, by passing that pointer to another thread).

Reply:
Yes, I agree, but it is necessary for one specific thing: Updating the loaded module list of a process. The tradeoff here is between having this function and writing an extra function that updates this list inside of DataManager. I decided for this tradeoff, but I am actually not sure anymore. Let me know what you think.

Reply:
I would argue for the function in data manager. But have no strong opinion on it. So I think it is fine for now. Thanks for the explanation.